Tests/Enquiry/Build/CheckYourAnswersTests.cs

```diff
@@ -86,7 +95,13 @@ public async Task With_a_valid_data_post_moves_to_next_page()
             SchoolId = 1,
             SchoolUrn = 123,
             Email = "test@test.com",
-            TutoringLogistics = "Test content",
+            TutoringLogisticsDetailsModel = new Application.Common.Models.Enquiry.TutoringLogisticsDetailsModel()
```

Comment:
By using the statement `using TutoringLogisticsDetailsModel = Application.Common.Models.Enquiry.TutoringLogisticsDetailsModel;` within the namespace, we create an alias for the fully qualified name `Application.Common.Models.Enquiry.TutoringLogisticsDetailsModel`. This alias allows us to refer to the class simply as `TutoringLogisticsDetailsModel()` instead of the complete namespace every time we need to instantiate an object of that class.

Reply:
Done